src/components/FlashMessages/FlashMessage.tsx

```diff
@@ -0,0 +1,66 @@
+// FlashMessage
```

Comment:
WDTY? Move this file into the `UI` folder?

Reply:
![](https://media.giphy.com/media/qDPg6HNz2NfAk/giphy.gif)